apps/namadillo/src/App/Common/ErrorBox.tsx

```diff
@@ -26,7 +26,7 @@ export const ErrorBox = ({
       <i className="text-4xl text-yellow">
         <MdErrorOutline />
       </i>
-      <p className="leading-[1.2]">{niceError}</p>
+      <div className="leading-[1.2]">{niceError}</div>
```

Comment:
Fix the console warning

![Screenshot 2024-08-07 at 11 44 23](https://github.com/user-attachments/assets/94d674fa-2454-4e1f-9cff-8f0d96dc4222)